SPECS/cloud-init/cloud-init.spec

```diff
@@ -16,10 +16,7 @@ Patch0:         cloud-init-azureds.patch
 Patch1:         ds-identify.patch
 Patch2:         ds-vmware-mariner.patch
 Patch3:         cloud-cfg.patch
-Patch4:         networkd.patch
-Patch5:         mariner.patch
-Patch6:         update-metadata-on-BOOT_NEW_INSTANCE.patch
-Patch7:         apply-netconfig-every-boot.patch
+Patch4:         mariner-21.4.patch
```

Comment:
Suggestion/Non-blocker: I'd maybe add more details to the file name or a comment above it, so it's clearer what this does.

Reply:
Good point. Added a comment!